packages/cli/lib/prompts/buildIdPrompt.js

```diff
@@ -0,0 +1,35 @@
+const { promptUser } = require('./promptUtils');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+
+const i18nKey = 'cli.lib.prompts.buildIdPrompt';
+
+const buildIdPrompt = (latestBuildId, deployedBuildId, projectName) => {
+  return promptUser({
+    name: 'buildId',
+    message: i18n(`${i18nKey}.enterBuildId`),
+    default: () => {
+      if (latestBuildId === deployedBuildId) {
+        return;
+      }
+      return latestBuildId;
+    },
+    validate: val => {
+      if (Number(val) > latestBuildId) {
+        return i18n(`${i18nKey}.errors.buildIdDoesNotExist`, {
+          buildId: val,
+          projectName,
+        });
```

Comment:
Per @mendel-at-hubspot, we could theoretically get into certain weird states where `latestBuild` is not the highest build ID, but practically it always will be. Figured it was worth it for the UX improvement, especially since using the flag is always an option.